src/animated/Controller.ts

```diff
@@ -90,13 +89,11 @@ class Controller<DS extends object = any> {
         // Eagerly create an Animated node to be used during render.
         // When `from[name]` exists, the node is created by `_diff` instead.
         if (!hasFrom && !this.values[name]) {
-          // Use undefined as the placeholder value
-          const nil = undefined as any
           const node = is.arr(value)
-            ? new AnimatedValueArray(fillArray(value.length, () => nil))
+            ? new AnimatedValueArray(value)
             : isAnimatableString(value)
-            ? new AnimatedValue(nil).interpolate({ output: [] })
-            : new AnimatedValue(nil)
+            ? new AnimatedValue(0).interpolate({ output: [value, value] })
+            : new AnimatedValue(value as any)
```

Comment:
No longer using a placeholder value like `undefined` or `NaN`.

Instead, the `to[name]` value is used as the initial value when `from[name]` is undefined.